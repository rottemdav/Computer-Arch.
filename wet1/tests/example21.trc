16 4 6 1 global_history global_tables using_share_lsb
0xa0f0 N 0x343fa8c
0x891b1c0 T 0x9a214
0xa3cd8 N 0xc28504d0
0xa0f0 N 0x343fa8c
0x891b1c0 T 0x9a214
0x891b1c0 T 0x9a214
0x84b8 N 0x5da7988
0xd64 T 0x419f9c60
0xa3cd8 T 0xc28504d0
0x9edaf28 T 0xfabb0
0x9edaf28 T 0xfabb0
0x84b8 N 0x5da7988
0x8fef64 N 0xe7ce874
0xa3cd8 N 0xc28504d0
0x9edaf28 N 0xfabb0
0x891b1c0 N 0x9a214
0x8fef64 T 0xe7ce874
0x8fef64 T 0xe7ce874
0xd64 T 0x419f9c60
0x9edaf28 T 0xfabb0
0x9edaf28 N 0xfabb0
0xd64 T 0x419f9c60
0x8fef64 N 0xe7ce874
0x8fef64 T 0xe7ce874
0xd64 T 0x419f9c60
0xa3cd8 T 0xc28504d0
0x84b8 N 0x5da7988
0x9edaf28 T 0xfabb0
0x8fef64 T 0xe7ce874
0xa3cd8 N 0xc28504d0
0x84b8 T 0x5da7988
0x891b1c0 T 0x9a214
0x9edaf28 T 0xfabb0
0x8fef64 N 0xe7ce874
0xd64 T 0x419f9c60
0x891b1c0 T 0x9a214
0x8fef64 N 0xe7ce874
0x8fef64 T 0xe7ce874
0x9edaf28 N 0xfabb0
0x9edaf28 N 0xfabb0
0xa3cd8 N 0xc28504d0
0x9edaf28 N 0xfabb0
0x84b8 N 0x5da7988
0xa0f0 N 0x343fa8c
0x8fef64 N 0xe7ce874
0xa3cd8 N 0xc28504d0
0x8fef64 N 0xe7ce874