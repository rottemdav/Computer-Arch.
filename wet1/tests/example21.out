0xa0f0 N 0xa0f4
0x891b1c0 N 0x891b1c4
0xa3cd8 N 0xa3cdc
0xa0f0 N 0xa0f4
0x891b1c0 N 0x891b1c4
0x891b1c0 N 0x891b1c4
0x84b8 N 0x84bc
0xd64 N 0xd68
0xa3cd8 N 0xa3cdc
0x9edaf28 N 0x9edaf2c
0x9edaf28 N 0x9edaf2c
0x84b8 T 0x5da7988
0x8fef64 N 0x8fef68
0xa3cd8 N 0xa3cdc
0x9edaf28 N 0x9edaf2c
0x891b1c0 T 0x9a214
0x8fef64 T 0xe7ce874
0x8fef64 N 0x8fef68
0xd64 N 0xd68
0x9edaf28 N 0x9edaf2c
0x9edaf28 N 0x9edaf2c
0xd64 N 0xd68
0x8fef64 N 0x8fef68
0x8fef64 N 0x8fef68
0xd64 N 0xd68
0xa3cd8 T 0xc28504d0
0x84b8 T 0x5da7988
0x9edaf28 N 0x9edaf2c
0x8fef64 N 0x8fef68
0xa3cd8 T 0xc28504d0
0x84b8 T 0x5da7988
0x891b1c0 T 0x9a214
0x9edaf28 N 0x9edaf2c
0x8fef64 N 0x8fef68
0xd64 N 0xd68
0x891b1c0 T 0x9a214
0x8fef64 N 0x8fef68
0x8fef64 T 0xe7ce874
0x9edaf28 T 0xfabb0
0x9edaf28 N 0x9edaf2c
0xa3cd8 N 0xa3cdc
0x9edaf28 N 0x9edaf2c
0x84b8 N 0x84bc
0xa0f0 N 0xa0f4
0x8fef64 T 0xe7ce874
0xa3cd8 N 0xa3cdc
0x8fef64 N 0x8fef68
flush_num: 23, br_num: 47, size: 628b
